pkg/broker/handler/processors/filter/processor.go

```diff
@@ -55,10 +57,24 @@ func (p *Processor) Process(ctx context.Context, event *event.Event) error {
 		return nil
 	}
 
+	trigger := types.NamespacedName{
```

Comment:
Do we need tracing on forwarding replied events?

Reply:
Event replies should propagate trace context over HTTP to the broker ingress, creating new broker and trigger spans.